failed to run local example

Without `--no-default-features`:
```
$ FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run
inpod_enabled: false
inpod_uds: /var/run/ztunnel/ztunnel.sock
inpod_port_reuse: true
inpod_mark: 1337

thread 'main' panicked at src/app.rs:299:5:
not implemented: fake_ca requires --features testing
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

With `--no-default-features`:
```
$ FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run --no-default-features
....
warning: unused import: `crate::tls::Error`
  --> src/tls/csr.rs:15:5
   |
15 | use crate::tls::Error;
   |     ^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

warning: unused import: `rustls::crypto::CryptoProvider`
  --> src/tls/lib.rs:25:5
   |
25 | use rustls::crypto::CryptoProvider;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no method named `generate` found for struct `CsrOptions` in the current scope
  --> src/identity/caclient.rs:64:10
   |
61 |           let cs = tls::csr::CsrOptions {
   |  __________________-
62 | |             san: id.to_string(),
63 | |         }
64 | |         .generate()?;
   | |         -^^^^^^^^ method not found in `CsrOptions`
   | |_________|
   |
   |
  ::: src/tls/csr.rs:22:1
   |
22 |   pub struct CsrOptions {
   |   --------------------- method `generate` not found for this struct
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following traits define an item `generate`, perhaps you need to implement one of them:
           candidate #1: `itertools::intersperse::IntersperseElement`
           candidate #2: `rand_core::block::BlockRngCore`
           candidate #3: `itertools::intersperse::IntersperseElement`
           candidate #4: `rand_core::block::BlockRngCore`

Some errors have detailed explanations: E0425, E0432, E0599.
For more information about an error, try `rustc --explain E0425`.
warning: `ztunnel` (lib) generated 2 warnings
error: could not compile `ztunnel` (lib) due to 6 previous errors; 2 warnings emitted

```

--- a/LOCAL.md
+++ b/LOCAL.md
@@ -16,7 +16,7 @@ There are a variety of config options that can be used to replace components wit
 * `NODE_NAME=local` - configures which node the ztunnel is running as.
   This impacts the networking path of requests. In the `localhost.yaml` example, `NODE_NAME=local` would make localhost use the in-memory fast path; without it HBONE would be used.
 
-Together, `FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run --features testing,tls-ring` (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
+Together, `FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run --features testing` (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
 
 ## Local in-pod testing
 

--- a/LOCAL.md
+++ b/LOCAL.md
@@ -16,7 +16,7 @@ There are a variety of config options that can be used to replace components wit
 * `NODE_NAME=local` - configures which node the ztunnel is running as.
   This impacts the networking path of requests. In the `localhost.yaml` example, `NODE_NAME=local` would make localhost use the in-memory fast path; without it HBONE would be used.
 
-Together, `FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run` (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
+Together, `FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run --features testing,tls-ring` (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
 
 ## Local in-pod testing
 

